core/src/main/java/nu/yona/server/subscriptions/service/UserAnonymizedDTO.java

```diff
@@ -44,6 +44,11 @@ public UUID getID()
 		return goals;
 	}
 
+	public String getTimeZoneId()
+	{
+		return "Europe/Paris";
```

Comment:
This needs to become modifiable. Please create a ticket for this. As we start in NL, let's use "Europe/Amsterdam" as the initial fixed value.